sigs/cncf-sigs.md

```diff
@@ -0,0 +1,252 @@
+
+# CNCF Special Interest Groups ("SIGs")
+
+Proposal by the CNCF TOC and Contributors
+Primary Authors: Alexis Richardson, Quinton Hoole
+November 2018 - January 2019
+
+Final Draft v1.0
+
+[[TOC]]
+
+## Overall Purpose
+
+Scale contributions by the CNCF technical and user community, while retaining integrity and increasing quality in support of our [mission](https://github.com/cncf/foundation/blob/master/charter.md#1-mission-of-the-cloud-native-computing-foundation). 
+
+## Specific Objectives
+
+* Strengthen the project ecosystem to meet the needs of end users and project contributors.
+
+* Identify gaps in the CNCF project portfolio.  Find and attract projects to fill these gaps.
+
+* Educate and inform users with unbiased, effective, and practically useful information.
+
+* Focus attention & resources on helping foster project maturity, systematically across CNCF projects.
+
+* Clarify relationship between projects, CNCF project staff, and community volunteers
+
+* Engage more communities and create an on-ramp to effective TOC contribution & recognition
+
+* Reduce some project workload on TOC while retaining executive control & tonal integrity with this elected body.
+
+* Avoid creating a platform for politics between vendors
+
+## Introduction
+
+A CNCF SIG will oversee and coordinate the interests pertaining to a logical area of needs of end users and/or projects.  Examples of such areas include security, testing, observability, storage, networking, etc.  The area overseen by a SIG is typically met by a set of CNCF projects, and may also represent a cross-cutting feature group shared by several projects (like security and observability).  SIG’s are:
+
+* long lived groups that report to the Technical Oversight Committee.
+
+* led primarily by recognised experts in the relevant field(s), supported by other contributors
+
+CNCF SIGs are modelled on Kubernetes SIGS.  Differences are intended to be minimal to avoid confusion - unavoidable differences are described [here](https://docs.google.com/document/d/1oSGhx5Hw7Hs_qawYB46BvRSPh0ZvFoxvHx-NWaf5Nsc/edit?usp=sharing).
+
+## Responsibilities & Empowerment of SIGs
+
+It is the desire of the TOC that the CNCF SIGs, under guidance from the TOC, provide high-quality technical expertise, unbiased information and proactive leadership within their category.  The TOC makes use of this input to act as an informed and effective executive board to select and promote appropriate CNCF projects and practices, and to disseminate high quality information to end users and the cloud-native community in general.  SIGs explicitly have no direct authority over CNCF projects. In particular, the creation of CNCF SIG’s does not change the existing, successfully practiced [charter](https://github.com/cncf/foundation/blob/master/charter.md) goal that "Projects.. will be ‘lightly’ subject to the Technical Oversight Committee". 
```

Comment:
`SIGs` or `SIG's`?

Reply:
Need clarification on how to ensure the information is unbiased. Should we publicize the input/information to the project/WG members who are doing the heavy-lifting work if the input/information is related to a project/WG?

Reply:
@NicolasT Agreed, done.
@cathyhongzhang Agreed.  Contributors and their company and project affiliations will be public.